### PR TITLE
[tempo] feat: add metrics generator to chart

### PR DESCRIPTION
**Pull request by nielstenboom**
Hi!

This PR adds support to the chart for the new [Tempo metrics generator](https://grafana.com/blog/2022/05/02/new-in-grafana-tempo-1.4-introducing-the-metrics-generator/#:~:text=The%20metrics%20generator%20is%20a,that%20supports%20Prometheus%20remote%20write.) introduced in 1.4 

relevant PR: https://github.com/grafana/tempo/pull/1282 
relevant issue: https://github.com/grafana/tempo/issues/1303

### CODE REVIEW

**Comment by BitProcessor** [comment]
@nielstenboom seems like something went wrong with the docs? The docs for `multitenancy_enabled` have disappeared?

**Comment by BitProcessor** [comment]
Screenshot 2022-07-18 at 15 42 23
Something is still wrong it seems, your version of the readme is pretty messed up ;)

File: charts/tempo/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Grafana Tempo Single Binary Mode
...
+| config | string | `"multitenancy_enabled: {{ .Values.tempo.multitenancyEnabled }}\nsearch_enabled: {{ .Values.tempo.searchEnabled }}\nmetrics_generator_enabled: {{ .Values.tempo.metricsGenerator.enabled }}\ncompactor:\n  compaction:\n    compacted_block_retention: {{ .Values.tempo.retention }}\ndistributor:\n  receivers:\n    {{- toYaml .Values.tempo.receivers | nindent 8 }}\ningester:\n  {{- toYaml .Values.tempo.ingester | nindent 6 }}\nserver:\n  {{- toYaml .Values.tempo.server | nindent 6 }}\nstorage:\n  {{- toYaml .Values.tempo.storage | nindent 6 }}\noverrides:\n  {{- toYaml .Values.tempo.global_overrides | nindent 6 }}\n  {{- if .Values.tempo.metricsGenerator.enabled }}\n      metrics_generator_processors: \n      - 'service-graphs'\n      - 'span-metrics'\nmetrics_generator:\n      storage:\n        path: \"/tmp/tempo\"\n        remote_write:\n          - url: {{ .Values.tempo.metricsGenerator.remoteWriteUrl }}\n  {{- end }}\n"` | Tempo configuration file contents |
```
Comment (BitProcessor) [inline charts/tempo/README.md:16]: This needs fixing. Results in:
Screenshot 2022-07-18 at 15 42 23

Comment (nielstenboom) [reply charts/tempo/README.md:16]: I think it's a feature 😄 

It's the same on the documentation that is currently in main: https://github.com/grafana/helm-charts/tree/main/charts/tempo#values

**Review by zanhsieh — APPROVED** [review]

**Review by BitProcessor — APPROVED** [review]
Thanks for your contribution! 🚀

**Pull request merged**